charts/consul/values.yaml

```diff
@@ -2185,10 +2185,10 @@ connectInject:
     # If this setting is false, you will need to install the Gateway API CRDs manually.
     manageExternalCRDs: true
 
-    # Enables Consul on Kubernets to manage only the custom CRDs used for Gateway API. If manageExternalCRDs is true 
-    # then all CRDs will be installed. If manageCustomCRDs is true then only TCPRoute, GatewayClassConfig and MeshService
+    # Enables Consul on Kubernets to manage only the non-standard CRDs used for Gateway API. If manageExternalCRDs is true 
+    # then all CRDs will be installed. If manageNonStandardCRDs is true then only TCPRoute, GatewayClassConfig and MeshService
```

Comment:
```suggestion
    # then all CRDs will be installed; otherwise, if manageNonStandardCRDs is true then only TCPRoute, GatewayClassConfig and MeshService
```